src/util/defs.h

```diff
@@ -22,6 +22,10 @@ constexpr unsigned int kMaxEngineSamples = kMaxEngineChannels * kMaxEngineFrames
 constexpr unsigned int MAX_BUFFER_LEN = 160000;
 
 constexpr int kMaxNumberOfDecks = 4;
+constexpr int kMaxNumberOfHotcues = 37;
```

Comment:
Btw why is this 37?
Some dev was bored of even numbers and 2^n and picked a prime number to make it a bit more exciting :rofl:

Reply:
This is for https://github.com/mixxxdj/mixxx/wiki/Stanton-Scs.3D#trig-modes
There must have happen a one-off isseue somewhere that we have 37 and not 36 = 3 x 12 
https://github.com/mixxxdj/mixxx/commit/622ed836b4578a472199e449ba1cd06679175430

Reply:
Okay, thanks for digging.
Shall we revert that to 36?
I know this would be considered a 'breaking change', but I wonder who is really using 37 hotcues?

Reply:
Yes, I support it.